res/translations/es.json

```diff
@@ -0,0 +1,230 @@
+    "Connecting to": "Conectando a",
+    "Content type": "Tipo de contenido",
+    "Could not reach server": "No se pudo conectar con el servidor",
+    "Create new playlist": "Crear nueva lista de reproducción",
+    "day": "día",
+    "days": "días",
+    "Delete Playlist": "Eliminar lista de reproducción",
+    "Demo": "Demo",
+    "Description": "Descripción",
+    "Disable server transcoding": "Desactivar transcodificación del servidor",
+    "Disc number": "Número de disco",
+    "Discography": "Discografía",
+    "discs": "discos",
+    "DJ-Mix": "Mezcla de DJ",
+    "Download": "Descargar",
+    "Download completed": "Descarga completa",
+    "Duration": "Duración",
+    "Edit": "Editar",
+    "Edit Playlist": "Editar lista de reproducción",
+    "Edit server": "Editar servidor",
+    "Enable system tray": "Habilitar bandeja del sistema",
+    "Enabled": "Habilitado",
+    "Enter": "Entrar",
+    "EP": "EP",
+    "Equalizer": "Ecualizador",
+    "Exclusive mode": "Modo exclusivo",
+    "Favorites": "Favoritos",
+    "Field Recording": "Grabación en campo",
+    "File path": "Ruta al archivo",
+    "File size": "Tamaño del archivo",
+    "Filter albums": "Filtrar álbumes",
+    "Filter genres": "Filtrar géneros",
+    "Forward": "Adelantar",
+    "Frequently Played": "Reproducidas frequentemente",
+    "General": "General",
+    "Genre": "Género",
+    "Genres": "Géneros",
+    "Github page": "Página de Github",
+    "Go to release page": "Ir a la página de lanzamiento",
+    "Hide": "Ocultar",
+    "Home": "Inicio",
+    "Home Page": "Página de inicio",
+    "hr": "hr",
+    "hrs": "hrs",
+    "Internet Radio Stations": "Estaciones de radio por Internet",
+    "Interview": "Entrevista",
+    "Is favorite": "Es favorito",
+    "Is not favorite": "No es favorito",
+    "Last played": "Última reproducción",
+    "Live": "En vivo",
+    "Locally": "Localmente",
+    "Log Out": "Cerrar sesión",
+    "Login to Server": "Iniciar sesión en el servidor",
+    "Lyrics": "Letras",
+    "Lyrics not available": "Letras no disponibles",
+    "Menu": "Menú",
+    "min": "min",
+    "minutes of track have been played": "minutos de la pista han sido reproducidos",
+    "Mixtape": "Mixtape",
+    "Mute": "Silencio",
+    "My Server": "Mi servidor",
+    "Name": "Nombre",
+    "Name (A-Z)": "Nombre (A-Z)",
+    "Next": "Siguiente",
+    "Nickname": "Apodo",
+    "No new version found": "No se encontró nueva versión",
+    "No radio stations available": "No hay estaciones de radio disponibles",
+    "None": "Ninguno",
+    "none": "ninguno",
+    "none selected": "ninguno seleccionado",
+    "OK": "OK",
+    "optional": "opcional",
+    "or when": "o cuando",
+    "Password": "Contraseña",
+    "Pause": "Pausar",
+    "Paused": "Pausada",
+    "Peak Meter": "Medidor de picos",
+    "percent of track is played": "porciento de la pista ha sido reproducida",
+    "Play": "Reproducir",
+    "Play Artist Radio": "Reproducir radio del artista",
```

Comment:
Play Artist Radio and Play Song Radio are using Spotify terminology  - if it makes more sense they could be translated like "Play Artist Mix" or "Play Similar Songs" but if radio makes sense in this context in Spanish it's good as-is

Reply:
I've changed both.